modules/installation-openshift-local.adoc

```diff
@@ -0,0 +1,18 @@
+// Module included in the following assemblies:
+//
+// * getting_started/openshift-overview.adoc
+// * installing/index.adoc
+
+:_content-type: CONCEPT
+[id="installation-openshift-local_{context}"]
+= {product-title} Local Overview
+
+Red Hat {product-title} Local (formerly Red Hat CodeReady Containers) supports rapid application development to get started building {product-title} clusters. {product-title} Local is designed to run on a local computer to simplify setup and testing, and to emulate the cloud development environment locally with all of the tools needed to develop container-based applications. 
+
+
+Whatever your programming language, {product-title} Local hosts your application and brings a minimal, preconfigured Red Hat {product-title} 4.1 or newer cluster to your local PC without the need for a server-based infrastructure.
+
+
+With {product-title} Local, you can create microservices, build them into images, and run them in Kubernetes-hosted containers, right on your laptop or desktop running Linux, macOS, or Windows 10. 
```

Comment:
`With` is weird, as `you can create microservices` and `build them into images` is not what we do. It is easily mis-read. We merely 'host' these applications

Reply:
Thanks for your review! I have updated this sentence to include some "hosting" language, please let me know if it makes sense or needs revision. Appreciate your time!